package/yast2-network.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Mon Feb 22 11:52:09 UTC 2016 - mfilka@suse.com
+
+- bnc#960703
+  - Moved setting network service into 1st stage
```

Comment:
This describes an implementation, but what was the problem? IIUC, "AY installation with NM leaves networking half-started, with empty resolv.conf"